antora-playbook.yml

```diff
@@ -80,9 +86,10 @@ content:
     - '!v2.5.0'
 ui:
   bundle:
-    url: https://github.com/opennms-forge/antora-ui-opennms/releases/download/v3.0.1/ui-bundle.zip
+    #url: https://github.com/opennms-forge/antora-ui-opennms/releases/download/v3.0.1/ui-bundle.zip
+    url: https://output.circle-artifacts.com/output/job/b282354c-4796-42e9-be8f-7a88bc920ed3/artifacts/0/ui-bundle.zip
```

Comment:
We have a dependency on the Antora UI update here: https://github.com/opennms-forge/antora-ui-opennms/pull/27